modules/fundamental/tests/cve.rs

```diff
@@ -0,0 +1,84 @@
+#![allow(clippy::expect_used)]
+
+use anyhow::bail;
+use futures_util::stream;
+use std::convert::Infallible;
+use test_context::test_context;
+use test_log::test;
+use trustify_common::id::Id;
+use trustify_module_ingestor::service::{Format, IngestorService};
+use trustify_test_context::TrustifyContext;
```

Comment:
For a test to belong in this directory, I think it should import something from this module. This test should probably live beneath `modules/ingestor/tests/`.

Reply:
Originally I was assuming to use stuff from `fundamental`, but it looks like that's not needed. So I'll move it.

Reply:
Moved.